resources/views/teams/details.blade.php

```diff
@@ -0,0 +1,34 @@
+
+@extends('layouts.app')
+
+@if(isset($team))
```

Comment:
I think we can safely assume that team is set. Este tipo de validaciones corresponden al Controlador y en todo caso no tendría mucho sentido llegar aquí sin un `team`

Reply:
Creo que una vez me dijiste que el front-end debería cargar incluso si me mandaran un PHP object vacío. Por eso lo he estado checando. ¿Aún así prefieres que quite esta validación?

Reply:
Sí, a lo que me refería es que atributos individuales del objeto pueden venir o no venir (dependiendo de si existen y están required) pero creo que el print de blade mostly care of that for you. En todo caso checar si existe la entidad es parte del trabajo del back-end. I think we can leave it off the front-end